MusicApp/MusicApp/Validations/AlbumValidation.cs

```diff
@@ -0,0 +1,83 @@
+﻿using FluentValidation;
+using MusicApp.Contracts;
+using Enums.AlbomTypes;
+using Utils;
+using DataBase.Models;
+using Microsoft.IdentityModel.Tokens;
+
+namespace MusicApp.Validations;
+
+public class AlbumCreateValidation : AbstractValidator<AlbumCreateContract>
+{
+    public AlbumCreateValidation()
+    {
+        RuleFor(x => x.Name)
+            .NotEmpty().WithMessage(Resources.Validations.AlbumNameCheckIsNotEmpty)
+            .Length(4, 30).WithMessage(Resources.Validations.AlbumNameCheckLength);
+
+        RuleFor(x => x.Release)
+            .NotEmpty().WithMessage(Resources.Validations.AlbumReleaseCheckIsNotEmpty)
+            .LessThan(x => DateTime.Now).WithMessage(Resources.Validations.AlbumReleaseShouldNotBeFuture);
+
+        RuleFor(x => x.Genres)
+            .Must(list => list.Count > 0 && list.Count < 10).WithMessage(Resources.Validations.AlbumGenresCheckIsNotEmpty);
+
+        RuleForEach(x => x.Genres)
+            .Length(3, 20).WithMessage(Resources.Validations.AlbumGenresCheckValidGenre);
+
+        RuleFor(x => x.Type)
+            .Must(x => x != AlbumTypes.Undefined).WithMessage(Resources.Validations.AlbumTypeCheckIsNotEmpty);
+    }
+}
+
+public class AlbumUpdateValidation : AbstractValidator<AlbumUpdateContract>
+{
+    public AlbumUpdateValidation()
+    {
+        RuleFor(x => x.Name)
```

Comment:
it looks pretty much the same as validation for the `create` operation. Think about consolidation of the models or introduce a hierarchy of the classes

Reply:
done